proxy/src/http/websocket.rs

```diff
@@ -181,8 +189,41 @@ async fn ws_handler(
 
         // Return the response so the spawned future can continue.
         Ok(response)
+    // TODO: that deserves a refactor as now this function also handles http json client besides websockets.
+    // Right now I don't want to blow up sql-over-http patch with file renames and do that as a follow up instead.
+    } else if request.uri().path() == "/sql" && request.method() == Method::POST {
+        let result = sql_over_http::handle(config, request, sni_hostname).await;
```

Comment:
We can also add timeout here, to limit extra large queries/responses the same way as it's done in the console.

Reply:
done